src/main/java/org/advent/daythree/Number.java

```diff
@@ -0,0 +1,72 @@
+package org.advent.daythree;
+
+import java.util.List;
+
+public class Number {
+    private final int numberValue;
+    private final int row;
+    private final int column;
+    private boolean isAdjacentToSymbol;
+    private StarCoordinates adjacentStarCoordinates;
+    private boolean hasBeenUsed;
+
+    public Number(int numberValue, int row, int column) {
+        this.numberValue = numberValue;
+        this.row = row;
+        this.column = column;
+    }
+
+    public boolean doesNumberHasAdjacentSymbol(List<String> listOfStrings, int rowIndex, int columnIndex, int numberLength) {
```

Comment:
why do you need rowIndex and columnIndex here?

https://github.com/fstetlerKnightec/AdventCalendar/pull/1#discussion_r1428825503